### ISSUE TITLE
Nested Headings causing "Content missing after heading"

### ISSUE THREAD

**Issue by GaryRidgway** [uiowa/uiowa#9300]
# Problem 
On the _Project_ pages for facilities, an `h1` tag is nested inside an `h1` tag without any content between, throwing a "Content missing after heading" accessibility error.

- [Here is a link to the SI page.](https://my2.siteimprove.com/Inspector/1005109/A11Y/Page?pageId=65057098265&impmd=6F4B1BAAA780A4D7A3CA00D93433A366&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r78/failed/bVFLeDBDUE8=;NWI2YWNkMmE=)
- [For convenience, here is the page in question.](https://facilities.uiowa.edu/project/1033001)

Here is a picture of the rendered HTML.

<img width="379" height="120" alt="Image" src="https://github.com/user-attachments/assets/43363b8a-7713-4678-a359-dc988acf187c" />

# Proposed solution

Find how it's rendered and remove one of the `h1` tags.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated - some investigation may be needed on where the heading is coming from. 

 **Discussion:**
 - Believe projects are imported content.

**Proposed solution or definition of done:** 
- Removal of one of the `<h1>` tags
- Verify SiteImprove no longer flagging as an issue.
